task2/src/App.jsx

```diff
@@ -1,50 +1,20 @@
-import {
-  MemoryRouter as Router,
-  Switch,
-  Route,
-  Link
-} from "react-router-dom";
-import { Hello, About, Home, Users } from '@components'
-  
+import styled from 'styled-components'
+import '@styles/roboto.css';
+import { Home } from '@pages/Home/Home'
+
+const AppStyled = styled.div`
+  background-color: #232323;
```

Comment:
JFYI: we store colors in separate root file and use them.

Reply:
> JFYI: we store colors in separate root file and use them.

I didn't think about that :(